docker/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Used for copy the self-signed certificate from caddy and add it to the trusted certificates
+# see https://github.com/dunglas/symfony-docker/blob/master/docs/troubleshooting.md#fix-chromebrave-ssl
+cert/*
```

Comment:
Do we need to create a temporary directory? Can't we use a pipe or a temporary file?

Reply:
It's now in tmp. Are these command good ?